build.gradle

```diff
@@ -5,6 +5,7 @@ subprojects {
     repositories {
       jcenter()
       maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+      mavenLocal()
```

Comment:
intentional?

Reply:
it helps to run integration tests inside IDE with locally installed new version (new changes) of plugin. But yeah will remove it.